lib/l10n/app_id.arb

```diff
@@ -174,24 +174,24 @@
   "done": "Selesai",
   "step_1": "Langkah 1",
   "first_go_to": "Pertama, Pergi ke",
-  "login_if_not_logged_in": "dan Masuk/Daftar jika Anda belum masuk",
+  "login_if_not_logged_in": "dan Masuk/Daftar jika kamu belum masuk",
   "step_2": "Langkah 2",
-  "step_2_steps": "1. Setelah Anda masuk, tekan F12 atau Klik Kanan Mouse > Buka Browser Devtools.\n2. Lalu buka Tab \"Aplikasi\" (Chrome, Edge, Brave, dll.) atau Tab \"Penyimpanan\" (Firefox, Palemoon, dll.)\n3. Buka bagian \"Cookie\" lalu subbagian \"https://accounts.spotify.com\"",
+  "step_2_steps": "1. Setelah kamu masuk, tekan F12 atau Klik Kanan Mouse > Buka Browser Devtools.\n2. Lalu buka Tab \"Aplikasi\" (Chrome, Edge, Brave, dll.) atau Tab \"Penyimpanan\" (Firefox, Palemoon, dll.)\n3. Buka bagian \"Cookie\" lalu subbagian \"https://accounts.spotify.com\"",
   "step_3": "Langkah 3",
   "step_3_steps": "Salin nilai Cookie \"sp_dc\" ",
   "success_emoji": "Berhasil🥳",
-  "success_message": "Sekarang Anda telah berhasil Masuk dengan akun Spotify Anda. Kerja bagus, sobat!",
+  "success_message": "Sekarang kamu telah berhasil Masuk dengan akun Spotify. Kerja bagus, sobat!",
   "step_4": "Langkah 4",
   "step_4_steps": "Tempel nilai \"sp_dc\" yang disalin",
   "something_went_wrong": "Terjadi kesalahan",
   "piped_instance": "Piped Server Instance",
   "piped_description": "The Piped server instance untuk digunakan sebagai pencocokan trek",
-  "piped_warning": "Beberapa di antaranya mungkin tidak berfungsi dengan baik. Jadi gunakan dengan risiko Anda sendiri",
+  "piped_warning": "Beberapa di antaranya mungkin tidak berfungsi dengan baik. Jadi gunakan dengan risiko dan tanggung jawabmu sendiri",
```

Comment:
`"piped_warning": "Beberapa di antaranya mungkin tidak berfungsi dengan baik. Jadi gunakan dengan risiko dan tanggung jawabmu sendiri",`
Let's make it more "human", How about 
`"piped_warning": "Mungkin ada beberapa yang tidak bekerja dengan baik. Jadi gunakan dengan hati-hati",`